src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1486,8 +1486,8 @@ private function getErrorMessages(Exception $e)
                 'Admin.Notifications.Error'
             ),
             InvalidOrderStateException::class => [
-                InvalidOrderStateException::UNEXPECTED_INVOICE => $this->trans(
-                    'Invalid action: this order already has an invoice.',
+                InvalidOrderStateException::ALREADY_PAID => $this->trans(
+                    'Invalid action: this order is already paid.',
```

Comment:
```suggestion
                    'Invalid action: this order has already been paid.',
```

Reply:
Damn.. that's what I originally used I was *this close* 👌 to have it right on the first try!! 😛

Reply:
Done